src/poliastro/coordinates.py

```diff
@@ -0,0 +1,169 @@
+"""Functions related to coordinate systems and transformations.
+
+This module complements :py:mod:`astropy.coordinates`.
+
+"""
+
+import math
+
+import astropy.time as time
+import astropy.coordinates as coordinates
+import astropy.units as u
+
+from poliastro.util import transform
+
+
+T = 0
```

Comment:
Now I understand what you told me. I don't like this approach either.

I would rather define one function for each body, receiving the `Time` as an input and then returning a pair `(ra, dec, W)` with the coordinates of the north pole and the prime meridian.